Zend/zend_compile.c

```diff
@@ -3864,6 +3863,24 @@ static zend_bool zend_is_constructor(zend_string *name) /* {{{ */
 }
 /* }}} */
 
+zend_function *zend_get_compatible_func_or_null(zend_class_entry *ce, zend_string *lcname) /* {{{ */
+{
+	zend_function *fbc = zend_hash_find_ptr(&ce->function_table, lcname);
+	if(!fbc || fbc->common.fn_flags & ZEND_ACC_PUBLIC || ce == CG(active_class_entry)){
```

Comment:
nit: `if (`, `) {`

Reply:
Also `(fbc->common.fn_flags & ZEND_ACC_PUBLIC)`. We prefer redundant parentheses for bitwise ops, because C has some really unintuitive precedence rules in that area...